src/zrc/conf/base.py

```diff
@@ -362,3 +363,11 @@
             'dsn': RAVEN_CONFIG['dsn']
         },
     })
+
+
+# settings for sending notifications
+NOTIFICATIES_URL = 'http://127.0.0.1:8001/api/v1/notificaties'
```

Comment:
this should be pulled from the environment with a sensible default (port 8004 I think)

Reply:
port was changes to 8004